includes/zigbee2mqtt/zigbee2mqtt.service

```diff
@@ -0,0 +1,15 @@
+[Unit]
+Description=zigbee2mqtt
```

Comment:
Maybe use the capitalized form of Zigbee2MQTT here as well as it is a user facing element.

Reply:
done